src/commands/run.js

```diff
@@ -266,12 +271,14 @@ exports.handler = function ({
             }
             const bin = getNodePackageManager()
             const startArguments = {
-              cwd: ctx.wrapperPath,
+              cwd: clientPath || ctx.wrapperPath,
               env: {
                 REACT_APP_ENS_REGISTRY_ADDRESS: ctx.ens,
                 BROWSER: 'none',
+                ARAGON_CLIENT_PORT: WRAPPER_PORT
```

Comment:
This needs to be renamed to `REACT_APP_PORT` as per https://github.com/aragon/aragon/pull/352#discussion_r216981196